lasy/profiles/from_openpmd_profile.py

```diff
@@ -75,8 +79,10 @@ def __init__(
         field,
         coord=None,
         is_envelope=None,
+        is_waket=None,
```

Comment:
For the record: we do this at the moment, but before the next release we need to make sure the code work without such a specific option.

Reply:
the `is_waket` flag is only needed to set the right filepath.
For some reason `lasy` assumes a certain file name scheme, e.g. prefix + "_%T.h5".
(see below).